src/py/flwr/cli/new/new_test.py

```diff
@@ -35,7 +35,7 @@ def test_render_template() -> None:
     """Test if a string is correctly substituted."""
     # Prepare
     filename = "app/README.md.tpl"
-    data = {"project_name": "FedGPT"}
+    data = {"project_name": "FedGPT", "module_name": "fedgpt", "import_name": "fedgpt"}
```

Comment:
Let's add one test case for each of the conversions and error situations handled by `flwr new`